paymentapi/src/main/java/fi/hel/verkkokauppa/payment/api/data/GetPaymentRequestDataDto.java

```diff
@@ -38,4 +39,12 @@ public String getLanguage() {
 	public void setLanguage(String language) {
 		this.language = language;
 	}
+
+	public String getMerchantId() {
```

Comment:
Could you add lombok `@getter and @setter` to class? No need to add these later on

Reply:
Good point! I'll add the change